os/arch/arm/src/imxrt/Kconfig

```diff
@@ -263,6 +297,23 @@ config IMXRT_LPSPI4
 
 endmenu # LPSPI Peripherals
 
+config IMXRT_I2S
+	bool "Audio I2S"
+	select I2S
+	select AUDIO
```

Comment:
It looks strange. Audio config can has I2S selection because AUDIO can not work without it. And AUDIO is high-level position module in architecture. But this code is that lowest level module enables higher level module. It looks strange.
And even though we provide drivers, someone does not want to use it and someone can use architecture driver directly. That is a choice of user. But this forces to use it.

Reply:
I agree with you.
I fixed it
Please check it